### DISCUSSION TITLE
Angular?

### DISCUSSION THREAD

**Discussion by tony** [zerobias/effector#735] [Q&A] [answered]
Will there be an examples / support for angular?  My story is I'm looking for a state system that works across frameworks 👍 

(did a search and haven't seen any topics yet!)

**Comment by AlexandrHoroshih** [comment, marked as answer]
Hi!

Effector stores are implementing the same Observable interface as RxJS does and events and effects can be called as any other function - so basic usage should be not much different from RxJS

There is a also case of [Fork API](https://effector.dev/docs/api/effector/fork) ([more detailed article](https://dev.to/effector/the-best-part-of-effector-4c27)), which requires special bindings ([e.g. `effector-react/scope` for example](https://effector.dev/docs/api/effector-react/useEvent)) - afaik no one have tried to create such bindings for Angular yet 🤔

**Reply by tony** [reply]
Thank you @AlexandrHoroshi!
